tests/calculators/geometry/test_calculate_angles.py

```diff
@@ -1,22 +1,13 @@
 import numpy as np
 import stko
 
+from .case_data import CaseData
 
-def test_calculate_angles(case_data):
-    """
-    Test :class:`.GeometryAnalyser.calculate_angles`.
-
-    Parameters:
-
-        case_data:
-            A test case.
-
-    """
 
+def test_calculate_angles(case_data: CaseData) -> None:
     analyser = stko.molecule_analysis.GeometryAnalyser()
 
     result = analyser.calculate_angles(case_data.molecule)
-    print(result)
```

Comment:
Mmmm, I like these print statements for when it fails... Is there a better way to do that?